app/repositories/submission_repository.rb

```diff
@@ -31,6 +31,14 @@ def waitlist
     rated_scope.offset(Setting.get.available_spots).to_a
   end
 
+  def to_invite
+    rated_scope
+      .where('confirmation_status <> ?', Submission.confirmation_statuses[:expired])
+      .limit(Setting.get.available_spots)
+      .where('confirmation_status = ?', Submission.confirmation_statuses[:not_available])
+      .to_a
```

Comment:
not sure if this `to_a` is needed here, without this it could be more flexible (you could add more conditions or something if you want)